docs/superbench-config.mdx

```diff
@@ -98,6 +106,22 @@ If not specified, will use [`${benchmark_name}.enable`](#enable) in each benchma
 * value from: benchmark names defined in `superbench.benchmarks`
 * default value: `null`
 
+### `superbench.monitor`
+
+Enable monitor to collect system metrics periodically, currently only support CUDA platform. There have three settings:
+
+#### `enable`
+
+Whether enable the monitor module or not.
+
+#### `sample_duration`
+
+Calculate the average metirc during sample_duration seconds, such as CPU usage and NIC bandwidth.
```

Comment:
is this average_duration instead of sample duration?

Reply:
Keep `sample_duration`, change `sample_freq` to `sample_interval`